polls.go

```diff
@@ -2,15 +2,17 @@ package main
 
 import (
 	"fmt"
+	"log"
+	"os"
+	"time"
+
 	"git_applet/gitter"
 	"git_applet/queries"
 	"git_applet/queries/aggregator"
+	"git_applet/queries/author"
 	"git_applet/queries/labeled"
 	"git_applet/queries/personal"
```

Comment:
what do you think of changing the packages to by<insertStrategyName>

Reply:
`by_author`, `by_label`, `by_repo`, `personal`?

Reply:
yes